README.md

```diff
@@ -25,16 +25,55 @@ git clone https://github.com/aerogear/mobile-developer-console && cd mobile-deve
 4. Log in with user `developer` (developer/123)
 5. Follow the link to mobile-developer-console
 
-### Full experience (Use APB to deploy mobile-developer-console)
+### Full experience
+
+#### Spin up OpenShift cluster locally
+
+**(OpenShift 3.9) Use mobile-core installer**
 
 1. Use [Mobile Core installer](https://github.com/aerogear/mobile-core) to run `oc cluster up` with
 all required configuration to deploy the mobile-developer-console from APB.
 You can follow the docs [here](https://github.com/aerogear/mobile-core)
-2. After your local OpenShift cluster is running, you can navigate to your project in OpenShift console and search for `Mobile Developer Console` in the catalog.
-3. Follow the wizard to provision Mobile Developer Console APB to your project
-4. Repeat steps 2 & 3 for `Mobile CI/CD` APB 
-5. Once both APBs are provisioned (provisioning of `Mobile CI/CD` will take couple of minutes), navigate the link to mobile-developer-console and log in as `developer`
+2. Follow the [steps below](#Provision-Mobile-Services)
+
+**(OpenShift 3.11) Use `oc cluster up` or `minishift`**
+
+:penguin: Linux
+See [OpenShift documentation](https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md) to learn how to spin up OpenShift 3.11 cluster locally using `oc cluster up` with following components enabled:
+* automation-service-broker
+* service-catalog
+* template-service-broker
+
+:apple: Mac
+Since `oc cluster up` is causing problems for users using Mac OS (since OpenShift version 3.10), it is advised to use Minishift as an alternative
+
+See [Minishift](https://docs.okd.io/latest/minishift/getting-started/index.html) documentation for more details
```

Comment:
similarly, it will be great if we can just give an example of the command to run here.

Reply:
could be something like this `MINISHIFT_ENABLE_EXPERIMENTAL=y minishift start --openshift-version v3.11.0 --extra-clusterup-flags "--enable=*,service-catalog,automation-service-broker,template-service-broker"`